content/en/faq/metrics-and-logs/telemetry-v1-vs-v2.md

```diff
@@ -22,6 +22,9 @@ v2 which are listed below:
   missing peer attributes like workload name, namespace and labels.
   However, if both peers have proxies injected all the labels mentioned
   [here](/docs/reference/config/metrics/) are available in the generated metrics.
+  At Istio 1.8 release, a solution was added to distribute server workload metadata
+  to client sidecar, which makes client side metric have server workload metadata labels
```

Comment:
```suggestion
  to the client sidecar causing client side metrics to have server workload metadata labels
```

Reply:
I suggest removing "Starting with the Istio 1.8 release, a solution was added" as I don't believe the history is important...especially in future releases. 

When the server workload is out of the mesh, server workload metadata is distributed to client sidecar, causing client side metrics to have server workload metadata labels filled.

Reply:
Updated to remove 1.8 release. thanks!